file_name: /home/uav/FYP_WS/src/my_offboard_node/traj/p2p.txt

control_freq: 100